recipes/moabs/meta.yaml

```diff
@@ -5,6 +5,7 @@ package:
   version: {{ version }}
 
 build:
+  skip: True  # [osx]
```

Comment:
Hi @dpryan79 , I realized you have added this `skip` option to build. May I ask what does this option intends to do? Thank you.

Reply:
It skips building on OSX, since it seems this doesn't deal with clang well.

Reply:
If it skips building on OSX, can this package be installed on OSX by conda?

Reply:
No